etc/profiles.yml

```diff
@@ -0,0 +1,19 @@
+# The valid profile names are:
+# "microsoft-azure", "google-cloud-platform", "amazon-web-services, "user_customized"
```

Comment:
a very small typo of the missing " mark here.

`# "microsoft-azure", "google-cloud-platform", "amazon-web-services, "user_customized"`

How about 

```
# The valid profile names are: microsoft-azure, google-cloud-platform, amazon-web-services, user_customized
# 
# NOTE: The `user_customized` profile will be loaded in the beginning as the default values, and parameters will be overriden by the correspondig profile environment if detected.

```

Reply:
Changed as
```
# The valid profile names are:
# "microsoft-azure", "google-cloud-platform", "amazon-web-services", "user_customized"
# Default is "user_customized" if no cloud environment detected
```